packages/react/src/primitives/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {
+  Root,
+  List,
+  Trigger as RadixTab,
+  Content as Panel,
+} from '@radix-ui/react-tabs';
+import { TabsProps, TabItemProps } from '../types';
+import { Flex } from '../Flex';
+import { ComponentClassNames } from '../shared/constants';
+import classNames from 'classnames';
+import { convertStylePropsToStyleObj, prefixer } from '../shared/utils';
+
+const isTabsType = (child: any): child is React.Component<TabItemProps> => {
```

Comment:
Maybe this function should be in the shared `util` file?

Reply:
I think if it is not used in any other file, it is fine to have it in this file only.